### PR TITLE
Add missing whitespace

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Add missing whitespace

```
ERROR: Found 2 pep8 issue(s) which need to be resolved:
--
1058 | 2023-01-04 10:00:26.447222 \| controller \| ERROR: plugins/module_utils/elbv2.py:48:8: E275: missing whitespace after keyword
1059 | 2023-01-04 10:00:26.447227 \| controller \| ERROR: plugins/module_utils/waf.py:140:20: E275: missing whitespace after keyword

```

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

plugins/module_utils/elbv2.py
plugins/module_utils/waf.py

### CODE REVIEW

File: plugins/module_utils/elbv2.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def _prune_ForwardConfig(action):
 
     # Remove the redundant ForwardConfig
     newAction = action.copy()
-    del(newAction["ForwardConfig"])
+    del (newAction["ForwardConfig"])
```
Comment (tremble) [inline plugins/module_utils/elbv2.py:48]:
```suggestion
    del newAction["ForwardConfig"]
```

File: plugins/module_utils/waf.py [hunk 2]
```diff
@@ -137,7 +137,7 @@ def get_rule(client, module, rule_id):
             if predicate['Type'] in match_sets:
                 predicate.update(match_sets[predicate['Type']](client, predicate['DataId']))
                 # replaced by Id from the relevant MatchSet
-                del(predicate['DataId'])
+                del (predicate['DataId'])
```
Comment (tremble) [inline plugins/module_utils/waf.py:140]:
```suggestion
                del predicate['DataId']
```

**Review by tremble — CHANGES_REQUESTED** [review]
I believe it's better to remove the extra () here...

File: plugins/module_utils/elbv2.py (hunk 1 above)
Comment (alinabuzachis) [reply plugins/module_utils/elbv2.py]: Done! Thanks!

**Review by tremble — APPROVED** [review]

**Pull request merged**